platform/security/common/src/main/java/ddf/security/common/util/Security.java

```diff
@@ -237,7 +237,23 @@ private static KeyStore getSystemKeyStore() {
                     e);
             return null;
         }
-        Path keyStoreFile = Paths.get(System.getProperty("javax.net.ssl.keyStore"));
+
+        // TODO: FIX ABSOLUTE PATH ASSIGNMENTS TO SYSTEM PROPERTIES
+        //      This is the "band-aid" for the problem so we can get a build running
+        //      This is not a permanent fix!!!
+        //  ------------------------------------------------------------------------
+        String propValue = System.getProperty("javax.net.ssl.keyStore");
+
+        // Instead of removing the ':', we remove the first slash IFF on windows
+        //  by computing a regular expression that detects the colon
+        propValue = propValue.replaceFirst("^/(.:/)", "$1");
+
+        //propValue = propValue.replace(":", "");
+        Path keyStoreFile = Paths.get(propValue);
+
+        //  ------------------------------------------------------------------------
+        //ORIGINAL CODE:
+        //Path keyStoreFile = Paths.get(System.getProperty("javax.net.ssl.keyStore"));
```

Comment:
We really shouldn't be doing any of this manually. The paths should be set correctly for the OS that you're using. We need to remove this code and instead determine the path to the keystore files. This isn't going to work for any of the code that bypasses this utility file (of which there is plenty).